packages/angular_devkit/build_webpack/src/webpack/index2.ts

```diff
@@ -53,7 +57,10 @@ export function runWebpack(
         // Log stats.
         log(stats, config);
 
-        obs.next({ success: !stats.hasErrors() });
+        obs.next({
+          success: !stats.hasErrors(),
+          emittedFiles: getEmittedFiles(stats.compilation),
```

Comment:
I am not sure if this is what we want, in general I would prefer, if the architect has a separate channel similar to progress, that we can send events too.

Something like `reportStatus`, but something more generic events that builders could report, this approach would also be useful for starting a server/listening on a port in devserver.
Something along
````
reportEvent('emittedFiles', {data});
reportEvent('serverStarted', {address, port, host});
````

Reply:
I agree.  Something similar would be beneficial.  We can move forward with this PR though and eventuallly deprecate as needed in the future when more extensive functionality is in place.